src/main/java/uk/gov/companieshouse/itemhandler/service/FilingHistoryDescriptionProviderService.java

```diff
@@ -81,7 +81,9 @@ public String mapFilingHistoryDescription(String descriptionKey, Map<String, Obj
             return null;
         }
         String description = getFilingHistoryDescriptionWithKey(descriptionKey);
-        if(filingHistoryDescriptionValues.containsKey("description")) {
+        if(filingHistoryDescriptionValues == null){
+            return description == null ? null : description.replace("*", "");
+        } if(filingHistoryDescriptionValues.containsKey("description")) {
```

Comment:
The `if` should go on to the next line.

Reply:
Yes this is supposed to be an `else if`, I'll add it in now